2023-12-22 11:59:44.695 - DBeaver 23.3.0.202312031432 is starting
2023-12-22 11:59:44.696 - OS: Windows 10 10.0 (amd64)
2023-12-22 11:59:44.696 - Java version: 17.0.6 by Eclipse Adoptium (64bit)
2023-12-22 11:59:44.697 - Install path: 'C:\Users\Class01\AppData\Local\DBeaver'
2023-12-22 11:59:44.697 - Instance path: 'file:/C:/dev/05-MySQL/'
2023-12-22 11:59:44.697 - Memory available 117Mb/1024Mb
2023-12-22 11:59:44.711 - Create display
2023-12-22 11:59:44.829 - Initializing DBeaver 23.3.0
2023-12-22 11:59:44.830 - Host plugin: org.jkiss.dbeaver.ui.app.standalone 23.3.0.202312031432
2023-12-22 11:59:44.830 - Initialize desktop platform...
2023-12-22 11:59:45.025 - BounceCastle bundle found. Use JCE provider BC
> Start Core File Systems [org.eclipse.core.filesystem 1.10.100.v20230717-1727]
> Start Core Resource Management [org.eclipse.core.resources 3.19.100.v20230814-1312]
> Start 팀 지원 핵심 [org.eclipse.team.core 3.10.100.v20230726-0617]
> Start DBeaver UI Navigator [org.jkiss.dbeaver.ui.navigator 1.0.120.202312031432]
2023-12-22 11:59:45.375 - Initialize base platform...
2023-12-22 11:59:45.403 - Platform initialized (573ms)
2023-12-22 11:59:45.453 - Starting instance server at http://localhost:51117
2023-12-22 11:59:45.462 - Run workbench
> Start DBeaver UI [org.jkiss.dbeaver.ui 5.1.135.202312031432]
> Start Eclipse e4 작업영역 SWT [org.eclipse.e4.ui.workbench.swt 0.17.100.v20230727-0604]
> Start Eclipse Application Services [org.eclipse.e4.core.services 2.4.100.v20230708-0809]
> Start Eclipse UI Application Services [org.eclipse.e4.ui.services 1.6.100.v20230630-1554]
> Start Eclipse e4 Progress View [org.eclipse.e4.ui.progress 0.4.100.v20230727-0604]
> Start Eclipse Bindings Support [org.eclipse.e4.ui.bindings 0.14.100.v20230630-1554]
> Start Eclipse e4 core commands [org.eclipse.e4.core.commands 1.1.100.v20230630-1554]
> Start EMF XML/XMI Persistence [org.eclipse.emf.ecore.xmi 2.35.0.v20230801-1141]
> Start Eclipse CSS SWT Theme Support [org.eclipse.e4.ui.css.swt.theme 0.14.100.v20230727-0604]
> Start Eclipse e4 작업영역 기능 추가 [org.eclipse.e4.ui.workbench.addons.swt 1.5.100.v20230727-0604]
Warning: NLS unused message: WorkbenchPreference_showMultipleEditorTabsButton in: org.eclipse.ui.internal.messages
Warning: NLS unused message: 이유 in: org.eclipse.ui.internal.messages
Warning: NLS unused message: Workbench_zoomChangedRestart in: org.eclipse.ui.internal.messages
> Start Eclipse e4 작업영역 SWT 렌더러 [org.eclipse.e4.ui.workbench.renderers.swt 0.16.100.v20230801-1255]
> Start Eclipse Dependency Injection Extensions [org.eclipse.e4.core.di.extensions 0.18.0.v20230429-1914]
> Start Eclipse CSS SWT Support [org.eclipse.e4.ui.css.swt 0.15.100.v20230727-0604]
> Start jna [com.sun.jna 5.13.0.v20230812-1000]
Warning: NLS unused message: BlockedJobsDialog_UserInterfaceTreeElement in: org.eclipse.ui.internal.progress.messages
2023-12-22 11:59:47.053 - Configure workbench window
> Start DBeaver SQL Model [org.jkiss.dbeaver.model.sql 1.0.110.202312031432]
2023-12-22 11:59:47.205 - Total database drivers: 108 (108)
> Start 도움말 시스템 코어 [org.eclipse.help 3.10.100.v20230726-0617]
2023-12-22 11:59:47.314 - Create actions
2023-12-22 11:59:47.322 - Disable Eclipse action set 'org.eclipse.ui.edit.text.actionSet.annotationNavigation'
2023-12-22 11:59:47.323 - Disable Eclipse action set 'org.eclipse.ui.WorkingSetActionSet'
2023-12-22 11:59:47.323 - Disable Eclipse action set 'org.eclipse.ui.actionSet.openFiles'
2023-12-22 11:59:47.774 - Initialize workbench window
2023-12-22 11:59:47.781 - Finish initialization
> Start 기본 문서 편집기 [org.eclipse.ui.editors 3.17.0.v20230821-1342]
> Start 텍스트 편집기 프레임워크 [org.eclipse.ui.workbench.texteditor 3.17.100.v20230821-0745]
> Start DBeaver UI Editors - Data [org.jkiss.dbeaver.ui.editors.data 1.0.120.202312031432]
Warning: NLS unused message: actions_navigator__objects in: org.jkiss.dbeaver.ui.editors.sql.internal.SQLEditorMessages
> Start DBeaver UI Editors - SQL [org.jkiss.dbeaver.ui.editors.sql 1.0.120.202312031432]
> Start DBeaver SQL Terminal [org.jkiss.dbeaver.ui.editors.sql.terminal 1.0.96.202312031432]
> Start 핵심 비교 지원 [org.eclipse.compare.core 3.8.100.v20230726-0617]
> Start 인터넷 연결 관리 사용자 인터페이스 [org.eclipse.ui.net 1.5.100.v20230726-0617]
> Start Internet Connection Management [org.eclipse.core.net 1.5.100.v20230630-1232]
> Start DBeaver Sample Database [org.jkiss.dbeaver.ext.sample.database 1.0.157.202312031432]
> Start DBeaver Tip of the day [org.jkiss.dbeaver.ext.ui.tipoftheday 1.0.126.202312031432]
> Start GEF Classic Draw2d [org.eclipse.draw2d 3.14.0.202308311129]
> Start GEF Classic GEF (MVC) [org.eclipse.gef 3.15.0.202308311129]
> Start DBeaver ERD UI [org.jkiss.dbeaver.erd.ui 3.0.80.202312031432]
> Start DBeaver ERD Model [org.jkiss.dbeaver.erd.model 1.0.80.202312031432]
> Start 콘솔 [org.eclipse.ui.console 3.13.0.v20230726-0617]
Warning: NLS unused message: InputStreamMonitor_label in: org.eclipse.debug.internal.core.DebugCoreMessages
Warning: NLS unused message: OutputStreamMonitor_label in: org.eclipse.debug.internal.core.DebugCoreMessages
> Start 디버그 핵심 [org.eclipse.debug.core 3.21.100.v20230731-1425]
2023-12-22 11:59:50.979 - Connect with 'bunnyroom 2' (mysql8-18c89e75609-74897c46972cb60b)
> Start DBeaver JDBC Model [org.jkiss.dbeaver.model.jdbc 1.0.13.202312031432]
> Start DBeaver MySQL Support [org.jkiss.dbeaver.ext.mysql 2.1.205.202312031432]
2023-12-22 11:59:51.019 - Execution context opened (bunnyroom 2; Main; 0)
> Start DBeaver GIS Support [org.jkiss.dbeaver.data.gis 1.0.144.202312031432]
2023-12-22 11:59:51.304 - Execution context opened (bunnyroom 2; Metadata; 1)
2023-12-22 11:59:51.482 - Connected (mysql8-18c89e75609-74897c46972cb60b, MySQL Connector/J [mysql-connector-j-8.2.0 (Revision: 06a1f724497fd81c6a659131fda822c9e5085b6c)])
2023-12-22 11:59:51.485 - Execution context opened (bunnyroom 2; SQLEditor <버니룸 테이블 생성 스크립트.sql>; 2)
2023-12-22 11:59:52.671 - Open [https://dbeaver.io/product/version.xml]
> Start DBeaver SQL Language Semantic Model [org.jkiss.dbeaver.model.lsm 1.0.15.202312031432]
> Start DBeaver SQL Server Model [org.jkiss.dbeaver.ext.mssql 1.0.214.202312031432]
> Start DBeaver SQLite Model [org.jkiss.dbeaver.ext.sqlite 1.0.169.202312031432]
> Start DBeaver Generic Plug-in [org.jkiss.dbeaver.ext.generic 2.3.206.202312031432]
> Start DBeaver Tasks - Native Tools UI [org.jkiss.dbeaver.tasks.native.ui 1.0.98.202312031432]
> Start DBeaver MySQL UI Support [org.jkiss.dbeaver.ext.mysql.ui 1.0.112.202312031432]
> Start DBeaver Tasks - General UI [org.jkiss.dbeaver.tasks.ui 1.0.98.202312031432]
2023-12-22 12:03:38.395 - Execution context opened (bunnyroom 2; SQLEditor <Console>; 3)
2023-12-22 12:04:24.222 - SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ADD COLUMN USER_WITHDRAW_REASON varchar(500)' at line 2
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ADD COLUMN USER_WITHDRAW_REASON varchar(500)' at line 2
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:607)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:510)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:517)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:971)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:4135)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5151)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ADD COLUMN USER_WITHDRAW_REASON varchar(500)' at line 2
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:767)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:652)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
2023-12-22 12:06:12.761 - Table 'columns' not found in metadata catalog
2023-12-22 12:06:12.761 - Table 'columns' not found in metadata catalog
2023-12-22 12:06:12.762 - Table 'columns' not found in metadata catalog
2023-12-22 12:06:48.893 - Table 'columns' not found in metadata catalog
2023-12-22 12:06:48.893 - Table 'columns' not found in metadata catalog
2023-12-22 12:06:48.893 - Table 'columns' not found in metadata catalog
2023-12-22 12:07:11.019 - Table 'columns' not found in metadata catalog
2023-12-22 12:07:11.019 - Table 'columns' not found in metadata catalog
2023-12-22 12:07:11.019 - Table 'columns' not found in metadata catalog
2023-12-22 12:07:28.030 - Table 'columns' not found in metadata catalog
2023-12-22 12:07:28.030 - Table 'columns' not found in metadata catalog
2023-12-22 12:07:28.030 - Table 'columns' not found in metadata catalog
2023-12-22 12:08:03.012 - Table 'columns' not found in metadata catalog
2023-12-22 12:08:03.013 - Table 'columns' not found in metadata catalog
2023-12-22 12:08:03.013 - Table 'columns' not found in metadata catalog
2023-12-22 12:09:33.291 - Table 'columns' not found in metadata catalog
2023-12-22 12:09:33.292 - Table 'columns' not found in metadata catalog
2023-12-22 12:09:33.292 - Table 'columns' not found in metadata catalog
2023-12-22 18:15:54.157 - 연결된 구성원으로부터 응답이 없어 연결하지 못했거나, 호스트로부터 응답이 없어 연결이 끊어졌습니다
java.net.SocketException: 연결된 구성원으로부터 응답이 없어 연결하지 못했거나, 호스트로부터 응답이 없어 연결이 끊어졌습니다
	at java.base/sun.nio.ch.NioSocketImpl.implRead(Unknown Source)
	at java.base/sun.nio.ch.NioSocketImpl.read(Unknown Source)
	at java.base/sun.nio.ch.NioSocketImpl$1.read(Unknown Source)
	at java.base/java.net.Socket$SocketInputStream.read(Unknown Source)
	at com.mysql.cj.protocol.ReadAheadInputStream.fill(ReadAheadInputStream.java:107)
	at com.mysql.cj.protocol.ReadAheadInputStream.readFromUnderlyingStreamIfNecessary(ReadAheadInputStream.java:150)
	at com.mysql.cj.protocol.ReadAheadInputStream.read(ReadAheadInputStream.java:180)
	at java.base/java.io.FilterInputStream.read(Unknown Source)
	at com.mysql.cj.protocol.FullReadInputStream.readFully(FullReadInputStream.java:64)
	at com.mysql.cj.protocol.a.SimplePacketReader.readHeaderLocal(SimplePacketReader.java:81)
	at com.mysql.cj.protocol.a.SimplePacketReader.readHeader(SimplePacketReader.java:63)
	at com.mysql.cj.protocol.a.SimplePacketReader.readHeader(SimplePacketReader.java:45)
	at com.mysql.cj.protocol.a.TimeTrackingPacketReader.readHeader(TimeTrackingPacketReader.java:52)
	at com.mysql.cj.protocol.a.TimeTrackingPacketReader.readHeader(TimeTrackingPacketReader.java:41)
	at com.mysql.cj.protocol.a.MultiPacketReader.readHeader(MultiPacketReader.java:54)
	at com.mysql.cj.protocol.a.MultiPacketReader.readHeader(MultiPacketReader.java:44)
	at com.mysql.cj.protocol.a.NativeProtocol.readMessage(NativeProtocol.java:576)
	at com.mysql.cj.protocol.a.NativeProtocol.checkErrorMessage(NativeProtocol.java:762)
	at com.mysql.cj.protocol.a.NativeProtocol.sendCommand(NativeProtocol.java:701)
	at com.mysql.cj.protocol.a.NativeProtocol.sendCommand(NativeProtocol.java:157)
	at com.mysql.cj.NativeSession.queryServerVariable(NativeSession.java:593)
	at com.mysql.cj.jdbc.ConnectionImpl.isReadOnly(ConnectionImpl.java:1356)
	at com.mysql.cj.jdbc.ConnectionImpl.isReadOnly(ConnectionImpl.java:1349)
	at com.mysql.cj.jdbc.ClientPreparedStatement.checkReadOnlySafeStatement(ClientPreparedStatement.java:291)
	at com.mysql.cj.jdbc.ClientPreparedStatement.execute(ClientPreparedStatement.java:301)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCPreparedStatementImpl.execute(JDBCPreparedStatementImpl.java:261)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCPreparedStatementImpl.executeStatement(JDBCPreparedStatementImpl.java:205)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.executeStatement(ExecuteBatchImpl.java:310)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.processBatch(ExecuteBatchImpl.java:178)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.execute(ExecuteBatchImpl.java:81)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.executeStatements(ResultSetPersister.java:822)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.lambda$2(ResultSetPersister.java:707)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.executeStatements(ResultSetPersister.java:706)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.run(ResultSetPersister.java:644)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-12-22 18:16:48.368 - Socket is closed.
java.io.IOException: Socket is closed.
	at com.mysql.cj.protocol.AbstractSocketConnection.getMysqlInput(AbstractSocketConnection.java:77)
	at com.mysql.cj.protocol.a.NativeProtocol.clearInputStream(NativeProtocol.java:865)
	at com.mysql.cj.protocol.a.NativeProtocol.sendCommand(NativeProtocol.java:682)
	at com.mysql.cj.protocol.a.NativeProtocol.sendCommand(NativeProtocol.java:157)
	at com.mysql.cj.NativeSession.queryServerVariable(NativeSession.java:593)
	at com.mysql.cj.jdbc.ConnectionImpl.isReadOnly(ConnectionImpl.java:1356)
	at com.mysql.cj.jdbc.ConnectionImpl.isReadOnly(ConnectionImpl.java:1349)
	at com.mysql.cj.jdbc.ClientPreparedStatement.checkReadOnlySafeStatement(ClientPreparedStatement.java:291)
	at com.mysql.cj.jdbc.ClientPreparedStatement.execute(ClientPreparedStatement.java:301)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCPreparedStatementImpl.execute(JDBCPreparedStatementImpl.java:261)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCPreparedStatementImpl.executeStatement(JDBCPreparedStatementImpl.java:205)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.executeStatement(ExecuteBatchImpl.java:310)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.processBatch(ExecuteBatchImpl.java:178)
	at org.jkiss.dbeaver.model.impl.data.ExecuteBatchImpl.execute(ExecuteBatchImpl.java:81)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.executeStatements(ResultSetPersister.java:822)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.lambda$2(ResultSetPersister.java:707)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.executeStatements(ResultSetPersister.java:706)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetPersister$DataUpdaterJob.run(ResultSetPersister.java:644)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-12-22 18:16:53.916 - Can't read execution warnings
org.jkiss.dbeaver.model.exec.DBCException: SQL Error [S1009]: No operations allowed after statement closed.
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.getStatementWarnings(JDBCStatementImpl.java:621)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:719)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:510)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:517)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:971)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:4135)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5151)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.sql.SQLException: No operations allowed after statement closed.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:98)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:90)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:64)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:74)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:82)
	at com.mysql.cj.jdbc.StatementImpl.getWarnings(StatementImpl.java:1739)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.getWarnings(JDBCStatementImpl.java:568)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.getStatementWarnings(JDBCStatementImpl.java:598)
	... 12 more
Caused by: com.mysql.cj.exceptions.StatementIsClosedException: No operations allowed after statement closed.
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.jdbc.StatementImpl.checkClosed(StatementImpl.java:340)
	at com.mysql.cj.jdbc.StatementImpl.getWarnings(StatementImpl.java:1724)
	... 14 more
2023-12-22 18:16:53.919 - Invalidate datasource 'bunnyroom 2' connections...
2023-12-22 18:16:53.921 - Invalidate datasource 'bunnyroom 2' (mysql8-18c89e75609-74897c46972cb60b)
2023-12-22 18:16:53.921 - 	Invalidate context 'Main' for mysql8-18c89e75609-74897c46972cb60b
2023-12-22 18:16:53.972 - Execution context closed (bunnyroom 2, 0)
2023-12-22 18:16:54.015 - 	Invalidate context 'Metadata' for mysql8-18c89e75609-74897c46972cb60b
2023-12-22 18:16:54.076 - Execution context closed (bunnyroom 2, 1)
2023-12-22 18:16:54.116 - 	Invalidate context 'SQLEditor <버니룸 테이블 생성 스크립트.sql>' for mysql8-18c89e75609-74897c46972cb60b
2023-12-22 18:16:54.167 - Execution context closed (bunnyroom 2, 2)
2023-12-22 18:16:54.207 - 	Invalidate context 'SQLEditor <Console>' for mysql8-18c89e75609-74897c46972cb60b
2023-12-22 18:16:54.258 - Execution context closed (bunnyroom 2, 3)
2023-12-22 18:16:54.322 - Operation failed. Retry count remains = 1
org.jkiss.dbeaver.model.sql.DBSQLException: SQL Error [08S01]: Communications link failure

The last packet successfully received from the server was 205,501 milliseconds ago. The last packet sent successfully to the server was 205,504 milliseconds ago.
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:133)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:607)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$1(SQLQueryJob.java:510)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:517)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:971)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:4135)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:123)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:190)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:121)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetViewer$ResultSetDataPumpJob.run(ResultSetViewer.java:5151)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet successfully received from the server was 205,501 milliseconds ago. The last packet sent successfully to the server was 205,504 milliseconds ago.
	at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:64)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:767)
	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:652)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.execute(JDBCStatementImpl.java:330)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCStatementImpl.executeStatement(JDBCStatementImpl.java:131)
	... 12 more
Caused by: com.mysql.cj.exceptions.CJCommunicationsException: Communications link failure

The last packet successfully received from the server was 205,501 milliseconds ago. The last packet sent successfully to the server was 205,504 milliseconds ago.
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Unknown Source)
	at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:104)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:149)
	at com.mysql.cj.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:165)
	at com.mysql.cj.protocol.a.NativeProtocol.clearInputStream(NativeProtocol.java:869)
	at com.mysql.cj.protocol.a.NativeProtocol.sendCommand(NativeProtocol.java:682)
	at com.mysql.cj.protocol.a.NativeProtocol.sendQueryPacket(NativeProtocol.java:1050)
	at com.mysql.cj.protocol.a.NativeProtocol.sendQueryString(NativeProtocol.java:997)
	at com.mysql.cj.NativeSession.execSQL(NativeSession.java:658)
	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:727)
	... 15 more
Caused by: java.io.IOException: Socket is closed.
	at com.mysql.cj.protocol.AbstractSocketConnection.getMysqlInput(AbstractSocketConnection.java:77)
	at com.mysql.cj.protocol.a.NativeProtocol.clearInputStream(NativeProtocol.java:865)
	... 20 more
2023-12-22 18:39:46.898 - Execution context closed (bunnyroom 2, 3)
2023-12-22 18:39:47.094 - Execution context closed (bunnyroom 2, 2)
2023-12-22 18:39:47.355 - DBeaver is stopping
2023-12-22 18:39:47.358 - Stop instance server
2023-12-22 18:39:48.356 - DBeaver shutdown completed
